src/EnergyPlus/WeatherManager.cc

```diff
@@ -2292,6 +2309,27 @@ namespace WeatherManager {
         if (state.dataEnvrn->EMSBeamSolarRadOverrideOn) state.dataEnvrn->BeamSolarRad = state.dataEnvrn->EMSBeamSolarRadOverrideValue;
         state.dataEnvrn->LiquidPrecipitation =
             state.dataWeatherManager->TodayLiquidPrecip(state.dataGlobal->TimeStep, state.dataGlobal->HourOfDay) / 1000.0; // convert from mm to m
+        if (state.dataEnvrn->RunPeriodEnvironment) {
```

Comment:
I wonder if this line should include `&& !Warmup` ?

Reply:
I'll add it in the && !Warmup. This part is used in reporting the precipitation-related quantities during the actual simulation period in the HTML tabular report. It should not include the warmup period.